iXBRLViewerPlugin/viewer/src/js/report.js

```diff
@@ -266,4 +266,17 @@ export class XBRLReport {
         }
         return this._calculationSummations.has(c);
     }
+
+
+    /**
+     * @return {Array[String]} Software credit text labels provided with this report for display purposes.
+     */
+    softwareCredits() {
+        if (!this._reportData.softwareCredits) {
```

Comment:
Is this going to throw an exception if the JS is used with a viewer that was generated with an older version of the python plugin that didn't include softwareCredits?